### ISSUE TITLE
Add information for Devs in README.md

### ISSUE THREAD

**Issue by czgdp1807** [codezonediitj/pydatastructs#96]
#### Description of the problem

Currently, the README.md has very little information for developers/contributors.
IMO, the following should be added after https://github.com/codezonediitj/pydatastructs#installation
```
Testing
----------
For testing your patch locally follow the steps given below,

1. Install [pytest-cov](https://pypi.org/project/pytest-cov/). Skip this step if you are already having the package.
2. Run, `python3 -m pytest --doctest-modules --cov=./ --cov-report=html`. Look for, `htmlcov/index.html` and open it in your browser, which will show the coverage report. Try to ensure that the coverage is not decreasing by less than 1% for your patch. 
```

**Comment by sagnik-chatterjee** [comment]
Hi , can i take up this issue ?

**Comment by annu12340** [comment]
I would like to work on this issue

**Comment by czgdp1807** [comment]
Hi @annu12340 
This has already been done by #97 
You can try #112 or make some improvements to the README on your own by adding information which you think should be there.

**Comment by annu12340** [comment]
ok

**Comment by czgdp1807** [comment]
Fixed by #97
